examples/tutorials/tutorial_1_aa_setup.m

```diff
@@ -0,0 +1,98 @@
+% Here we create a base parameter xml file, that will contain your default
+% settings.
+% It will be used each time aarecipe is called without explicitly passing a
+% parameter filename.
+% Settings can be overriden later where needed, in specific parameter files
+% and in run scripts, but that is not a subject in this tutorial file.
+aas_create_parameter_xml('', true, 'use_default_location', true, 'use_default_filename', true);
+
+%% ------------------------------------------------------------------------
+% Creating an aap structure
+% -------------------------------------------------------------------------
+% Call aarecipe to createa the aap settings structure from a parameter file
+% and a tasklist file.
+% The default parameter file created in the previous step will be used,
+% since no parameter filename is provided.
+% The helloworld tasklist comes installed with aa:
+[aahome,~,~] = fileparts(which('aa_ver5'));
+tasklist_fname = fullfile(aahome,'examples' ,'tutorials', 'tutorial_1_aa_setup.xml');
+aap = aarecipe(tasklist_fname);
+
+%% ------------------------------------------------------------------------
+% Specifying the results directories
+% -------------------------------------------------------------------------
+% Results will be created in the directory specified in
+% aap.acq_details.root, in a specific subdirectory as specified in
+% aap.directory_conventions.analysisid
+% It is possible within a run script to override the default values taken
+% from the parameter file.
+%
+% The directory in aap.acq_details.root must exist on your machine
+% When it runs, aa will create a subdirectory RESULTS_DIR.
+% aap.acq_details.root = ROOT_PATH; % Outcommented: use value from parameter file
+RESULTS_DIR = mfilename(); % Typically specific for a particular analysis.
+aap.directory_conventions.analysisid = RESULTS_DIR;
+
+%% ------------------------------------------------------------------------
+% Specifying the data directory
+% -------------------------------------------------------------------------
+% For BIDS data, point rawdatadir at the top level BIDS directory
+% (i.e., wherever you downloaded ds000114)
+FULLDATAPATH = '/full/path/to/toplevelBIDS';
+aap.directory_conventions.rawdatadir = FULLDATAPATH;
+
+%% ------------------------------------------------------------------------
+% Using BIDS input
+% -------------------------------------------------------------------------
+% When using BIDS input, run aas_processBIDS before aa_doprocessing
+%
+% Here we only run sub-01 for the finger_foot_lips task (ds000114
+% contains four different tasks) since we're just testing whether
+% your aa install is working.
+
+aap = aas_processBIDS(aap, [], {'finger_foot_lips'}, {'sub-01'});
+
+%% ------------------------------------------------------------------------
+% Run
+% -------------------------------------------------------------------------
+aa_doprocessing(aap);
+aa_close(aap);
```

Comment:
Users typically do not run `aa_close` manually, because it is called by `aa_doprocessing`. Users have to run `aa_close` only if aa crashes and they want to restart the analysis.